config.js

```diff
@@ -44,8 +44,9 @@ const config = {
       '/xr_design_and_accessibility',
     ],
     collapsedNav: ['/tutorials'],
-    links: [{ text: 'Pluto', link: 'https://pluto.app' }],
-    frontline: false,
+    collapsedNav: ['/platforms'],
+    links: [{ text: 'CONTACT US', link: 'mailto:mait@plutovr.com' }],
+    frontline: true,
```

Comment:
What does this do? I tried to do a google and failed miserably. 😭

Reply:
`frontline` that is

Reply:
from `README.MD` 😉: 
![image](https://user-images.githubusercontent.com/10997806/144942974-acf33a38-132f-4499-88ed-c9af2793e13b.png)

but, don't feel too bad - the option is misspelled _in the template_ (capitalized differently in `config.js`). Furthermore turning it on simply kills the submenus. Life is complicated.